libraries/rush-sdk/package.json

```diff
@@ -10,6 +10,17 @@
   "homepage": "https://rushjs.io",
   "main": "lib-shim/index.js",
   "typings": "dist/rush-lib.d.ts",
+  "exports": {
+    ".": "./lib-shim/index.js",
+    "./loader": "./lib-shim/loader.js"
+  },
+  "typesVersions": {
+    "*": {
+      "loader": [
+        "./dist/loader.d.ts"
+      ]
+    }
+  },
```

Comment:
```suggestion
  "exports": {
    ".": {
      "default": "./lib-shim/index.js",
      "types": "dist/rush-lib.d.ts"
    }
    "./loader": {
      "default": "./lib-shim/loader.js",
      "types": "./dist/loader.d.ts"
    }
  },
```

Reply:
IIRC that syntax is only compatible with very recent TypeScript versions